paprika/build.py

```diff
@@ -36,10 +65,12 @@ def write_tleapin(
         elif pbctype == 2:
             f.write("solvateoct {} {} {:0.5f} iso\n".format(
                 unit, waterbox, bufferval))
+        elif pbctype is None:
+            f.write("# Skipping solvation ...\n")
```

Comment:
Is this permanent or temporary? If latter, probably mark as such.

Reply:
What do you mean by "this"?  1 --> True?  Yeah I think the latter is better.  Not sure what you mean by "mark".

Reply:
This: no solvation if `pbctype` is `None`.

Reply:
Permanent.  For a gas phase prmtop/inpcrd, I want to use the `write_tleap` function, and I don't want to be forced to do a solvatebox/oct command.